jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PluginConfigurationProcessor.java

```diff
@@ -124,6 +135,32 @@ static PluginConfigurationProcessor processCommonConfiguration(
         fromCredential);
   }
 
+  /**
+   * Get the user specified entrypoint or the default one
```

Comment:
The description isn't really correct, because the code may try to infer a main class. (For example, it may read `MANIFEST.MF` in a JAR.)

Reply:
Done